drivers/spi/Kconfig.xec_qmspi_v2_ldma

```diff
@@ -0,0 +1,18 @@
+# Microchip XEC QMSPI V2, QMSPI LDMA
+
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "Select Microchip XEC QMSPI driver"
+	default SPI_XEC_QMSPI_LDMA
+	depends on SPI && SOC_SERIES_MEC172X
+	help
+	 Select Microchip XEC QMSPI driver implementation.
+
+config SPI_XEC_QMSPI_LDMA
+	bool "Use QMSPI driver with LDMA (local dma)"
```

Comment:
add:
```
default y
depends on DT_HAS_MICROCHIP_XEC_QMSPI_LDMA_ENABLED
```

Reply:
replaced with choice in spi_xec_qmspi_full_duplex.h
could you check that?